docs/source/tutorials/arch_modeling/spypads_tutorial.rst

```diff
@@ -0,0 +1,332 @@
+GTKWave. 
+
+.. note:: To read about the differences between the **formal testbench** and the **full testbench**, please visit our page on testbenches: :ref:`testbench`.
+
+To open the task configuration file, run this command from the root directory of OpenFPGA:
+
+.. code-block:: bash
+
+   emacs openfpga_flow/tasks/fpga_verilog/spypad/config/task.conf
+
+The last line of the task configuration file (**LINE44**) sets the **formal testbench** to be the desired testbench. To use the **full testbench**, comment out **LINE44**.
+The file will look like this when finished:
+
+.. code-block:: python
+   :linenos:
+   
+   # = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
+   # Configuration file for running experiments
+   # = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
+   # timeout_each_job : FPGA Task script splits fpga flow into multiple jobs
+   # Each job execute fpga_flow script on combination of architecture & benchmark
+   # timeout_each_job is timeout for each job
+   # = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
+
+   [GENERAL]
+   run_engine=openfpga_shell
+   power_tech_file = ${PATH:OPENFPGA_PATH}/openfpga_flow/tech/PTM_45nm/45nm.xml
+   power_analysis = true
+   spice_output=false
+   verilog_output=true
+   timeout_each_job = 20*60
+   fpga_flow=vpr_blif
+
+   [OpenFPGA_SHELL]
+   openfpga_shell_template=${PATH:OPENFPGA_PATH}/openfpga_flow/openfpga_shell_scripts/example_script.openfpga
+   openfpga_arch_file=${PATH:OPENFPGA_PATH}/openfpga_flow/openfpga_arch/k6_frac_N10_adder_register_scan_chain_depop50_spypad_40nm_openfpga.xml
+   openfpga_sim_setting_file=${PATH:OPENFPGA_PATH}/openfpga_flow/openfpga_simulation_settings/auto_sim_openfpga.xml
+
+   [ARCHITECTURES]
+   arch0=${PATH:OPENFPGA_PATH}/openfpga_flow/vpr_arch/k6_frac_N10_tileable_adder_register_scan_chain_depop50_spypad_40nm.xml
+
+   [BENCHMARKS]
+   bench0=${PATH:OPENFPGA_PATH}/openfpga_flow/benchmarks/micro_benchmark/and2/and2.blif
+   # Cannot pass automatically. Need change in .v file to match ports
+   # When passed, we can replace the and2 benchmark
+   #bench0=${PATH:OPENFPGA_PATH}/openfpga_flow/benchmarks/micro_benchmark/test_mode_low/test_mode_low.blif
+
+   [SYNTHESIS_PARAM]
+   bench0_top = and2
+   bench0_act = ${PATH:OPENFPGA_PATH}/openfpga_flow/benchmarks/micro_benchmark/and2/and2.act
+   bench0_verilog = ${PATH:OPENFPGA_PATH}/openfpga_flow/benchmarks/micro_benchmark/and2/and2.v
+
+   #bench0_top = test_mode_low
+   #bench0_act = ${PATH:OPENFPGA_PATH}/openfpga_flow/benchmarks/micro_benchmark/test_mode_low/test_mode_low.act
+   #bench0_verilog = ${PATH:OPENFPGA_PATH}/openfpga_flow/benchmarks/micro_benchmark/test_mode_low/test_mode_low.v
+   bench0_chan_width = 300
+
+   [SCRIPT_PARAM_MIN_ROUTE_CHAN_WIDTH]
+   end_flow_with_test=
+   #LINE44 vpr_fpga_verilog_formal_verification_top_netlist=

```

Comment:
Since you have shown the line number, there is no need to specify the LINE44 here. I am cool if the syntax highlight is not shown. Line number is enough to direct readers.